rdr_service/data_gen/generators/study_nph.py

```diff
@@ -236,7 +239,11 @@ def create_nph_ordered_sample(self, order_id: int, parent_sample_id: Optional[in
             "status": status,
         }
         nph_ordered_sample = OrderedSample(**nph_ordered_sample_params)
-        return self.nph_ordered_sample_dao.insert(nph_ordered_sample)
+        nph_ordered_sample_dao = NphOrderedSampleDao()
+        with nph_ordered_sample_dao.session() as session:
+            session.add(nph_ordered_sample)
+            session.commit()
+            return nph_ordered_sample
```

Comment:
I cannot do it because, the `insert_with_session` defined on `NphOrderedSampleDao` expects `order` Namespace object which is causing failures. Hence, I had to make the modification for generating the test data.